File: backend/src/main/java/com/se114p12/backend/dto/request/general/NotificationRequest.java
package com.se114p12.backend.dto.request.general;

import com.se114p12.backend.enums.NotificationType;
import lombok.Data;
import org.springframework.lang.Nullable;

import java.util.List;

@Data
public class NotificationRequest {
    private List<Long> userIds;
    private NotificationType type;
    private String title;
    private String message;
    @Nullable
    private Integer status;
}

File: frontend/MAM/app/src/main/java/com/example/mam/dto/notification/NotificationRequest.kt
package com.example.mam.dto.notification

import androidx.annotation.Nullable
import com.example.mam.entity.Notification

data class NotificationRequest(
    val usersIds: List<Long> = listOf(),
    val type: String = "",
    val title: String = "",
    val message: String = "",
    val status: Int? = 0
)
